Sample_v3/Extensions/Channel+MembersInfo.swift

```diff
@@ -4,12 +4,19 @@
 
 import StreamChatClient
 
+/// Creates formatted string for channel members and online members.
```

Comment:
nit: Since this is a sample app, we can explain to people what's happening here so they don't have to read the code:

```swift
/// Creates a formatted string for channel members and online members.
///
/// Example result: "15 members, 3 online"
```

WDYT?

Reply:
Nice, done!